config/releases.exs

```diff
@@ -0,0 +1,24 @@
+# File: project_tides/config/releases.exs
```

Comment:
please add some info about what this file do. Will be good for newcomers/first-time-contributors to understand.